### ISSUE TITLE
Use SVG images to avoid pixelation

### ISSUE THREAD

**Issue by Atreyu-94** [mickstar/Zoo-For-Zotero#99]
I know it's not a must, but the app icons get pixelated. If you decide to use the SVG format, I would be happy to help shape the icons in the new format, obviously based on those of Zotero.
You would need the dimensions of the icons, file names, color palette and we feed back over here. :D

**Comment by mickstar** [comment]
Sure, the graphics i'm using are just copy pasted from the zotero desktop application. https://github.com/mickstar/Zoo-For-Zotero/tree/master/app/src/main/res/drawable
If you want to upgrade them to svg i will include them in the app.

**Comment by Atreyu-94** [comment]
They are many!! Hahaha. But I'll start doing it as soon as possible :)

**Comment by Atreyu-94** [comment]
Well, making a new set of consistent icons is difficult. Can we use the KDE Breeze icon set? The repository is this: https://github.com/KDE/breeze-icons
If you ever decide to include a dark theme, for example, this set allows for a dark mode. I also think it has a compatible license, LGPL.
Tonight I will start looking for icons according to the project. Please let me know if you agree to continue implementing it.

**Comment by Atreyu-94** [comment]
Hey! At last I used the icons from the Zotero Web Library.
https://github.com/zotero/web-library/tree/master/src/static/icons/16/item-types/light/1x
I renamed them, adjusted the dimensions to the desktop and at least the items are already complete in the dimensions of the desktop application.
I'm still missing the ones from the EPUB and DjVU files that I'm designing. I don't think there are any licensing problems either. I leave a screenshot. Regards.
![Captura de pantalla_2020-09-07_09-25-48](https://user-images.githubusercontent.com/39834909/92392896-fc0ed500-f0ec-11ea-84f9-1952d24b9876.png)
